### PR TITLE
Add zsh completion for '--cluster-store-opt'

### PR DESCRIPTION
**Pull request by sdurrheimer**
Zsh completion for `docker daemon --cluster-store-opt~`
ref. #16644

@thaJeztah Thanks for the ping

@vincentbernat 

Signed-off-by: Steve Durrheimer s.durrheimer@gmail.com

### CODE REVIEW

File: contrib/completion/zsh/_docker [hunk 1]
```diff
@@ -426,8 +426,9 @@ __docker_subcommand() {
                 "($help -D --debug)"{-D,--debug}"[Enable debug mode]" \
                 "($help)--default-gateway[Container default gateway IPv4 address]:IPv4 address: " \
                 "($help)--default-gateway-v6[Container default gateway IPv6 address]:IPv6 address: " \
-                "($help)--cluster-store=-[URL of the distributed storage backend]:Cluster Store:->cluser-store" \
+                "($help)--cluster-store=-[URL of the distributed storage backend]:Cluster Store:->cluster-store" \
```
Comment (thaJeztah) [inline contrib/completion/zsh/_docker]: oh, lol, thanks for catching that

**Comment by thaJeztah** [comment]
Thanks @sdurrheimer !

**Comment by vincentbernat** [comment]
LGTM!

**Comment by thaJeztah** [comment]
Thanks for reviewing @vincentbernat !

ping @tianon @jfrazelle for the merge

**Comment by icecrime** [comment]
Thanks! LGTM

**Pull request merged**